components/tikv_util/src/logger/file_log.rs

```diff
@@ -230,12 +193,117 @@ impl Rotator for RotateBySize {
     }
 }
 
+pub enum Task {
+    Archive,
+}
+
+impl Display for Task {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(f, "")
```

Comment:
You can adds `#[derive(Debug)]` attribute to `Task` and use it.
```rust
write!(f, "{:?}", self)
```

Reply:
Fixed, thanks!